### PR TITLE
#160451455 User should be able to reset password

### PR DESCRIPTION
**Pull request by kampkelly**
What does this PR do?
This PR add functionality to reset a users password.

Description of Task to be completed?
add forgot password page
add reset password page
#### How should this be manually tested?
Go to the forgot password and reset password page.
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#160451455

### CODE REVIEW

**Comment by kampkelly** [comment]
Not ready for review until login feature is merged

File: src/containers/Password/ForgotPassword.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+   */
+
+  /**
+   * @description - This method runs when the component upodates
+   * @returns {string} response
+   * @memberof ForgotPassword
+   */
+  componentDidUpdate() {
+    if (this.props.loading) {
+      if (this.props.errorMessage) {
```
Comment (kampkelly) [inline src/containers/Password/ForgotPassword.js]: this is the error message that shows if the forgot password email was sent successfully or not

File: src/containers/Password/ResetPassword.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+  /**
+   * @description - This method sends the forgot password request
+   * @returns {object} null
+   * @memberof ForgotPassword
+   */
+  onClickResetPassword() {
+    const password = $('#password').val();
+    const confirmPassword = $('#confirmPassword').val();
+    const { token } = this.props.match.params;
+    if (password.length < 6) {
```
Comment (kampkelly) [inline src/containers/Password/ResetPassword.js]: this checks that the new password must be at least 6 letters

File: src/containers/Login/Login.scss [hunk 3]
```diff
@@ -1,7 +1,5 @@
-#login-modal{
-
+#login-modal, #reset-password-modal, #forgot-password-modal{
```
Comment (s-akinrele) [inline src/containers/Login/Login.scss]: give the elements a className

File: src/containers/Password/ResetPassword.js [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+            <Input type="password" id="confirmPassword" label="Confirm Password" s={12} />
+            <Button
+              id="reset-button" waves='light' onClick={this.onClickResetPassword.bind(this)}>
+              Reset Password
+            </Button>
+          </Row>
+        </form>
+        <h6>
+          Have an account?
+          <a href="#" onClick={this.onLoginClicked.bind(this)}>
```
Comment (s-akinrele) [inline src/containers/Password/ResetPassword.js]: Always bind in the constructor

**Review by s-akinrele — CHANGES_REQUESTED** [review]
I had some comments

**Review by s-akinrele — APPROVED** [review]

**Pull request merged**